### PR TITLE
Remove parsing of skaffold file

### PR DESCRIPTION
**Pull request by samlesu**
This mode is unused and it doesn't make sense to maintain it.
It's gonna be harder to support it when the network configuration will be more complex.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ You will need to install it thanks to the `pip` binary.
 
 # Run the tests
 
-## Using a network configuration file
+The network configuration must be defined in a yaml file.
```
Comment (AurelienGasser) [inline README.md]: - What about "is" instead of "must be"?
- What about "described" instead of "defined"?

File: README.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ You will need to install it thanks to the `pip` binary.
...
 
-This is the mode used by default. Two files are currently available:
+Two configurations are currently available:
```
Comment (AurelienGasser) [inline README.md]: What about "configuration files" instead of "configurations"?

File: README.md (hunk 1 above)
Comment (samlesu) [reply README.md]: Thanks.

File: README.md (hunk 2 above)
Comment (samlesu) [reply README.md]: Thanks.

**Comment by samlesu** [comment]
> I agree with @AurelienGasser's comments regarding the wording, but it's otherwise good for me!

👍 
Thanks and done.

**Comment by samlesu** [comment]
Rebased after recent merges

**Pull request merged**